content/authors/Ngopnang/_index.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
It looks like this file is empty again. When you are editing the file on GitHub make sure you are putting the content into the file itself and _not_ into the comment area. The comment area is only for describing the change you are making.